src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/TotrinnskontrollService.kt

```diff
@@ -50,12 +54,25 @@ class TotrinnskontrollService(
             ) {
                 "Kan ikke sende til beslutter da det eksisterer en totrinnskontroll med status=${eksisterandeTotrinnskontroll.status}"
             }
+        } else {
+            feilHvis(sendTilBeslutterRequest.kommentarTilBeslutter != null) {
+                "Kan ikke legge ved kommentar til beslutter dersom behandlingen ikke er tidligere underkjent"
+            }
         }
+
+        behandlingshistorikkService.opprettHistorikkInnslag(
+            behandlingId = saksbehandling.id,
+            stegtype = saksbehandling.steg,
+            utfall = null,
+            metadata = sendTilBeslutterRequest,
```

Comment:
Burde man kanskje ta med de felter man ønsker fra `sendTilBeslutterRequest`, i tillfelle man legger til andre felter i requesten som ikke burde være i historikken? 👀 
Eks
`mapOf("kommentar" to sendTilBeslutterRequest.kommentar)` (eller riktig feltnavn)

Burde egentlige gjøre det på andre steder og